src/main/webapp/WEB-INF/context/portlet/simpleJspPortletContext.xml

```diff
@@ -22,11 +22,17 @@
 <beans xmlns="http://www.springframework.org/schema/beans"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xmlns:context="http://www.springframework.org/schema/context"
+        xmlns:util="http://www.springframework.org/schema/util"
         xsi:schemaLocation="
                 http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
-                http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context-2.5.xsd">
+                http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context-2.5.xsd
+                http://www.springframework.org/schema/util http://www.springframework.org/schema/util/spring-util.xsd">
 
     <context:annotation-config/>
-    <bean class="org.jasig.portlet.widget.mvc.SimpleJspPortletController"/>
+    <bean class="org.jasig.portlet.widget.mvc.SimpleJspPortletController">
+        <property name="properties">
+            <util:properties id="appLauncherProperties" location="classpath://simple-jsp.properties"/>
```

Comment:
minor - id is not correctly named.  Can probably just be removed.  I can do that post merge.